### DISCUSSION TITLE
parent wire:submit not working when nesting lazy livewire component

### DISCUSSION THREAD

**Discussion by BiazPrasastha** [livewire/livewire#5982] [7. Bugs (archived)]
![image](https://github.com/livewire/livewire/assets/49233008/9c996b1a-e01c-42dc-9e45-a19ffc776ce6)
Uncaught ReferenceError: message is not defined
    added http://livewire.test/livewire/livewire.js?id=578b80d0:8147
    patchElement http://livewire.test/livewire/livewire.js?id=578b80d0:7503
    patch http://livewire.test/livewire/livewire.js?id=578b80d0:7473
    morph http://livewire.test/livewire/livewire.js?id=578b80d0:7686
    morph2 http://livewire.test/livewire/livewire.js?id=578b80d0:8107
    <anonymous> http://livewire.test/livewire/livewire.js?id=578b80d0:8178
    VoidFunction* http://livewire.test/livewire/livewire.js?id=578b80d0:8177
    trigger http://livewire.test/livewire/livewire.js?id=578b80d0:173
    processEffects http://livewire.test/livewire/livewire.js?id=578b80d0:434
    handleResponse http://livewire.test/livewire/livewire.js?id=578b80d0:413
    succeed http://livewire.test/livewire/livewire.js?id=578b80d0:289
    succeed http://livewire.test/livewire/livewire.js?id=578b80d0:289
    sendRequestToServer http://livewire.test/livewire/livewire.js?id=578b80d0:270
    queueNewRequestAttemptsWhile http://livewire.test/livewire/livewire.js?id=578b80d0:313
    sendRequestToServer http://livewire.test/livewire/livewire.js?id=578b80d0:207
    triggerSend http://livewire.test/livewire/livewire.js?id=578b80d0:193
    requestBufferTimeout http://livewire.test/livewire/livewire.js?id=578b80d0:201
    setTimeout handler*bundleMultipleRequestsTogetherIfTheyHappenWithinFiveMsOfEachOther http://livewire.test/livewire/livewire.js?id=578b80d0:200
    triggerSend http://livewire.test/livewire/livewire.js?id=578b80d0:192
    requestCall http://livewire.test/livewire/livewire.js?id=578b80d0:350
    waitUntilTheCurrentRequestIsFinished http://livewire.test/livewire/livewire.js?id=578b80d0:307
    waitUntilTheCurrentRequestIsFinished http://livewire.test/livewire/livewire.js?id=578b80d0:303
    requestCall http://livewire.test/livewire/livewire.js?id=578b80d0:348
    <anonymous> http://livewire.test/livewire/livewire.js?id=578b80d0:3813
    anonymous http://livewire.test/livewire/livewire.js?id=578b80d0 line 896 > AsyncFunction:3
    generateEvaluatorFromString http://livewire.test/livewire/livewire.js?id=578b80d0:914
    tryCatch http://livewire.test/livewire/livewire.js?id=578b80d0:838
    observer3 http://livewire.test/livewire/livewire.js?id=578b80d0:5033
    observer3 http://livewire.test/livewire/livewire.js?id=578b80d0:5030
    src_default5 http://livewire.test/livewire/livewire.js?id=578b80d0:5029
    flushHandlers http://livewire.test/livewire/livewire.js?id=578b80d0:996
    stopDeferring http://livewire.test/livewire/livewire.js?id=578b80d0:1001
    deferHandlingDirectives http://livewire.test/livewire/livewire.js?id=578b80d0:1004
    initTree http://livewire.test/livewire/livewire.js?id=578b80d0:1187
    start http://livewire.test/livewire/livewire.js?id=578b80d0:1142
    start http://livewire.test/livewire/livewire.js?id=578b80d0:1141
    start3 http://livewire.test/livewire/livewire.js?id=578b80d0:7911
    <anonymous> http://livewire.test/livewire/livewire.js?id=578b80d0:8755
    EventListener.handleEvent* http://livewire.test/livewire/livewire.js?id=578b80d0:8754
    <anonymous> http://livewire.test/livewire/livewire.js?id=578b80d0:8758
livewire.js:8147:15
Uncaught TypeError: "Snapshot missing on Livewire component with id: " is not a constructor
    Component http://livewire.test/livewire/livewire.js?id=578b80d0:3828
    initComponent http://livewire.test/livewire/livewire.js?id=578b80d0:3876
    start3 http://livewire.test/livewire/livewire.js?id=578b80d0:7900
    skipDuringClone http://livewire.test/livewire/livewire.js?id=578b80d0:1592
    initTree http://livewire.test/livewire/livewire.js?id=578b80d0:1190
    initTree http://livewire.test/livewire/livewire.js?id=578b80d0:1190
    walk http://livewire.test/livewire/livewire.js?id=578b80d0:1113
    initTree http://livewire.test/livewire/livewire.js?id=578b80d0:1188
    deferHandlingDirectives http://livewire.test/livewire/livewire.js?id=578b80d0:1003
    initTree http://livewire.test/livewire/livewire.js?id=578b80d0:1187
    start http://livewire.test/livewire/livewire.js?id=578b80d0:1135
    onMutate http://livewire.test/livewire/livewire.js?id=578b80d0:668
    onMutate http://livewire.test/livewire/livewire.js?id=578b80d0:668
    MutationCallback* http://livewire.test/livewire/livewire.js?id=578b80d0:557
    <anonymous> http://livewire.test/livewire/livewire.js?id=578b80d0:8758
livewire.js:3828:15

**Comment by BiazPrasastha** [comment]
![image](https://github.com/livewire/livewire/assets/49233008/49d486cf-220c-44de-8045-3a72e8d64fbe)

**Comment by PhiloNL** [comment]
Hey @BiazPrasastha,
Thanks for reporting. Would you be able to PR a failing test for this?

**Comment by alecpl** [comment]
I'm seeing this error too, using 3.0.8. Looking at the code, no wonder. The `message` variable is not defined anywhere. https://github.com/livewire/livewire/blob/b9eca7d888669e54579db9a3f9f87e2ed53b1c3b/js/morph.js#L83
